portal-api/test-helpers/mock-utilisation-reports.js

```diff
@@ -0,0 +1,16 @@
+const MOCK_UTILISATION_REPORT = {
+  bank: {
+    id: '123',
+    name: 'test bank',
+  },
+  month: 1,
+  year: 2021,
+  dateUploaded: new Date('2023-11-07'),
+  path: 'test path',
+  uploadedBy: {
+    id: '123',
+    name: 'test user',
+  },
+};
```

Comment:
Considering your use case of this mock, you overwrite `month` and `year` but not `dateUploaded`. It may be more worthwhile to instead make a function that generates a report given a month and year. This will then make sure the `dateUploaded` matches what you would expect it to be and doesn't cause any issues down the line with those two fields not matching up